spotdl/download/downloader.py

```diff
@@ -298,6 +298,11 @@ def set_id3_data(self, convertedFilePath, songObj):
             desc='Cover',
             data=rawAlbumArt
         )
+        #! setting the lyrics
```

Comment:
```suggestion
        # ! setting the lyrics
```

![image](https://user-images.githubusercontent.com/42355410/110792353-8f2ef000-8273-11eb-9ea7-56ccafb5e4d1.png)